recastai/apis/build/build.py

```diff
@@ -26,7 +26,12 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
   @token_required
-  def dialog(self, msg, conversation_id, language=None):
+  def dialog(self, msg, conversation_id, language=None, **options):
+
+    memory = {} if options.get('memory') is None else memory = options.get('memory')
```

Comment:
The second `memory =` is not necessary and confusing :) You can write :
`memory = {} if options.get('memory') is None else options.get('memory')